bin/chunk_assembly_for_vecscreen.py

```diff
@@ -1,36 +1,45 @@
 #!/usr/bin/env python3
 """
-Script for chunking an assembly before running NCBI VecScreen. Adapted from a script by James Torrance.
-The script was further refactored by Eerik Aunin and Yumi Sims
+Script for chunking an assembly before running NCBI VecScreen. Adapted from a script by James Torrance, edited by Eerik Aunin and Yumi Sims
```

Comment:
Let's add Damon's name here as well, as he edited the script too

Reply:
When testing this new version of the script with the `PlasmoDB-66_Pfalciparum3D7_Genome.fasta` assembly (`/lustre/scratch123/tol/teams/tola/users/ea10/pipeline_testing/20231120_vecscreen_chunk_test/PlasmoDB-66_Pfalciparum3D7_Genome.fasta`), there is still some small difference between the output of this script and the old script from James.
This script: 25262914 bp, 55 sequences `/lustre/scratch123/tol/teams/tola/users/ea10/pipeline_testing/20231120_vecscreen_chunk_test/PlasmoDB-66_Pfalciparum3D7_Genome.chunks.yumi_v2.fasta`
James's script: 25182839 bp, 53 sequences
`/lustre/scratch123/tol/teams/tola/users/ea10/pipeline_testing/20231120_vecscreen_chunk_test/PlasmoDB-66_Pfalciparum3D7_Genome.chunks.james.fasta`
But this small difference probably doesn't matter for VecScreen